mypy/config_parser.py

```diff
@@ -278,6 +280,13 @@ def parse_section(prefix: str, template: Options,
                 if 'follow_imports' not in results:
                     results['follow_imports'] = 'error'
         results[options_key] = v
+        if key in ('files', 'packages', 'modules'):
+            if all(('packages' in results.keys() or 'modules' in results.keys(),
+                    results.get('files'))):
+                print("May only specify one of: module/package, files, or command. Ignoring key.",
```

Comment:
What does 'command' here refer to?

Please include the name of the key in the error message. Currently it may not be clear what is being ignored.

Reply:
Good point, and I'll remove command, that's part of the corresponding CLI error message that I preserved but doesn't apply here.